### PR TITLE
Add recipe for pytest-run-parallel

### PR DESCRIPTION
**Pull request by lysnikolaou**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [X] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by lysnikolaou** [comment]
@andfoy I've added you as a maintainer here. Please confirm that you're okay with that.

**Comment by lysnikolaou** [comment]
@conda-forge/help-python this is ready for review! Thanks!

**Comment by lysnikolaou** [comment]
@conda-forge-admin, please ping conda-forge/help-python

File: recipes/pytest-run-parallel/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/pytest_run_parallel-{{ version }}.tar.gz
+  sha256: accc9cd7eb298acbdd7d864ce3a38daaa45ab6a92e12b9c5db81b65bb19f609f
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
```
Comment (ocefpaf) [inline recipes/pytest-run-parallel/meta.yaml:14]:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation 
```

File: recipes/pytest-run-parallel/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    - python >={{ python_min }}
+    - pytest >=6.2.0
+
+test:
+  imports:
+    - pytest_run_parallel
+  commands:
+    - pip check
+  requires:
+    - pip
```
Comment (ocefpaf) [inline recipes/pytest-run-parallel/meta.yaml:32]:
```suggestion
    - python {{ python_min }}
    - pip
```

**Pull request merged**